bin/halcyon-build

```diff
@@ -10,9 +10,17 @@ private_storage_setup() {
 [ -f ./.env ] && source ./.env
 
 if ! private_storage_setup; then
-  echo 'You must set the Halcyon variables for AWS credentials' >&2
-  echo 'https://halcyon.sh/reference/#private-storage-options' >&2
-  exit 1
+  cat >&2 <<EOF
```

Comment:
Yeah, I guess. I did consider that option, but ultimately decided against. I don't feel strongly though, so I'll go ahead with your suggestion.